### PR TITLE
Update 2017_10

### PR DESCRIPTION
**Pull request by maxime-allex**

### CODE REVIEW

File: content/user/content/manipulate/lasso.md [hunk 1]
```diff
@@ -8,6 +8,8 @@ Select `Toggle lasso`.
 Move the lasso around the nodes you are interested in selecting to 
 select them.
 
+> You can also toggle lasso by simply clicking on the graph with the key `ctrl` or `cmd` (on MacOS) and move your mouse to create the lasso.
```
Comment (davidrapin) [inline content/user/content/manipulate/lasso.md]: You can also use the lasso by pressing the `ctrl` key (`cmd` on Mac OS) while dragging with the mouse to draw the outline of the desired selection.

File: content/user/content/manipulate/layouts.md [hunk 2]
```diff
@@ -27,6 +24,9 @@ Such layouts position nodes according to their connections: connected
 nodes are usually closed to each others, while disconnected nodes are
 usually pushed further.
 
+By setting "incremantal expand" active, force-directed layouts will be
```
Comment (davidrapin) [inline content/user/content/manipulate/layouts.md]: By enabling "incremental expand", [...]

File: content/user/content/manipulate/layouts.md [hunk 3]
```diff
@@ -60,3 +57,23 @@ Will position root nodes at the bottom of the screen.
 
 **Right to left Mode:**
 Will position root nodes at the right side of the screen.
+
+## Radial layout
+
+Radial layouts positions nodes around the selected one based on their
```
Comment (davidrapin) [inline content/user/content/manipulate/layouts.md]: The radial layout positions nodes around the currently selected node (used as center of the layout) based on their theoretical distance (shortest path in the graph). [...]

File: content/user/content/manipulate/layouts.md [hunk 4]
```diff
@@ -60,3 +57,23 @@ Will position root nodes at the bottom of the screen.
...
+
+## Radial layout
+
+Radial layouts positions nodes around the selected one based on their 
+graph-theoretical distance (shortest path in the graph, connecting them).
+This is useful to reveal layers in data and to draw the rest of the 
+graph in its relation to the pre-defined focus node.
+
+**Best mode:**
+Will use energy model and produce more readable layouts. It is also capable
```
Comment (davidrapin) [inline content/user/content/manipulate/layouts.md]: Will use *an* energy model *to* produce more readable layouts. [...]

File: content/user/content/manipulate/layouts.md [hunk 5]
```diff
@@ -60,3 +57,23 @@ Will position root nodes at the bottom of the screen.
...
+graph in its relation to the pre-defined focus node.
+
+**Best mode:**
+Will use energy model and produce more readable layouts. It is also capable 
+of handling special cases like disconnected components.
+
+![](Radial.png)
+
+**Fast mode:**
+Will use geometrical model and is faster but produce a lot of overlapping edges.
```
Comment (davidrapin) [inline content/user/content/manipulate/layouts.md]: Will use a geometrical model, which is faster but can produce more overlapping edges.

File: content/user/content/manipulate/undo.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
+
+You now have the possibility to revert your last action on a visualization.
+
+> You can revert only the last action you do on the visualization.
+
+For example, after expanding a node, clicking on the undo button 
+(or using the shortcut `ctrl-z` or `cmd-z` on MacOS) will collapse and remove all nodes added by the expand.
```
Comment (davidrapin) [inline content/user/content/manipulate/undo.md]: - don't use `collapse`, it means something else in LKE.
- clicking the "undo" button (or [...]) will revert the expand by returning the graph to its previous state.

File: content/user/content/manipulate/undo.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+
+![](Undo.png)
+
+After reverting the expand, you can go back and re-run it by clicking on the redo button (or using the shortcut `ctrl-y` or `cmd-y` on MacOS).
+
+![](Redo.png)
+
+You can undo all actions that impact your visualization except when you :
```
Comment (davidrapin) [inline content/user/content/manipulate/undo.md]: Some actions **cannot** currently be undone:
- Changing the captions
- Changing the styles
- Creating a node (or edge)
- Deleting a node (or edge)

Comment (maxime-allex) [reply content/user/content/manipulate/undo.md]: Removed "Creating a node (or edge)"

Comment (davidrapin) [reply content/user/content/manipulate/undo.md]: @maxime-allex please replace with this version

**Pull request merged**